### ISSUE TITLE
pattern versioning

### ISSUE THREAD

**Issue by charlieroberts** [kitzeller/barbara#4]
I'm not sure if this is how you conceptualize variants, but it seems like maybe they're forks of patterns? What about being able to see all the various iterations of a pattern created by a single author? It would be awesome to be able to explicitly fork a pattern and also to just save new iterations. Perhaps you have to fork before you can edit another user's pattern? This is kinda what Glitch does if I remember correctly.

Basically, I think the variants view is cool (although I'm a bit unclear what qualifies as a variant... is it like a fork?) and would like to see it extended to cover minor revisions / changes to a pattern.

**Comment by charlieroberts** [comment]
Trying to be clearer:

1. If you make a new drawing, there should be a File > Save As... option
2. If you start editing an existing drawing by another user, there should be a File > Fork (maybe Clone? Borrow? Copy? Fork seems a little techno-jargony) menu item
3. If you are editing your own drawing, there are File > Save and File > Save As... options.
